tmexp/preprocess.py

```diff
@@ -190,7 +190,11 @@ def feature_extractor(uast_obj: Any) -> Iterator[Tuple[str, str]]:
             lang_count[lang] += 1
             seen_files.add((file_path, blob_hash))
         files_info[ref][file_path] = {"blob_hash": blob_hash, "language": lang}
-    logger.info("Found %d parsable blobs:" % raw_count)
+    if raw_count:
+        logger.info("Found %d parsable blobs:" % raw_count)
+    else:
+        logger.info("Found no parsable blobs, stopping.")
```

Comment:
maybe replace with a RunTime Error ? so as to be consistent, since we always raise an error when something unexpected happens

Reply:
I'm not sure about this one: in my case at least it was expected behavior that some repos did not contain parsable blobs (Markdown only stuff, etc), and returning an error was “wrong” in the sense that preprocessing worked as intended. WDYT?

Reply:
ah I see, yeah makes sense